libs/ledger-live-common/src/hw/actions/initSell.ts

```diff
@@ -1,188 +0,0 @@
-import { Observable, of, concat } from "rxjs";
-import { scan, tap, catchError } from "rxjs/operators";
-import { useEffect, useState } from "react";
-import type { ConnectAppEvent, Input as ConnectAppInput } from "../connectApp";
-import type { Action, Device } from "./types";
-import type { Transaction, TransactionStatus } from "../../generated/types";
-import type { AppState } from "./app";
-import { log } from "@ledgerhq/logs";
-import { createAction as createAppAction } from "./app";
-import type { InitSellResult, SellRequestEvent } from "../../exchange/sell/types";
```

Comment:
If this action is not used, i  think `InitSellResult` can be removed. But I don't have enough knowledge on those, it's up to PTX to decide

Reply:
Oh well `libs/ledger-live-common/src/exchange/sell/types.ts` is removed too ahah 🧹